benchmarks/src/main/scala/akka/grpc/HandlerProcessingBenchmark.scala

```diff
@@ -30,7 +30,7 @@ class HandlerProcessingBenchmark extends CommonBenchmark {
   val in = Source.repeat(ServerReflectionRequest()).take(10000)
   val request: HttpRequest = {
     implicit val serializer = ServerReflection.Serializers.ServerReflectionRequestSerializer
-    GrpcRequestHelpers(Uri("https://unused.example/" + ServerReflection.name + "/ServerReflectionInfo"), in)
+    GrpcRequestHelpers(Uri("https://unused.example/" + ServerReflection.name + "/ServerReflectionInfo"), Nil, in)
```

Comment:
looks like the `benchmarks` project is not being compiled by travis/github actions

Reply:
oops! seems like the build the `benchmark-java` subproject but not the `benchmarks`, indeed